src/index.html

```diff
@@ -0,0 +1,32 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Leaderboard</title>
+</head>
+<body>
+    <h1>Leaderboard</h1>
+    <main>
+        <div class="left-main">
+            <div class="recent">
+                <h3>Recent scores</h3>
+                <button>Refresh</button>
+            </div>
+            <ul id="scores">
+                <li>Nabil: 100</li>
+                <li>Nabil: 100</li>
+                <li>Nabil: 100</li>
+                <li>Nabil: 100</li>
+                <li>Nabil: 100</li>
+            </ul>
+        </div>
+        <div class="inputs">
+            <h3>Add your score</h3>
+            <input type="text" placeholder="Your name">
+            <input type="text" placeholder="Your score">
+            <button>Submit</button>
+        </div>
```

Comment:
- [ ]  Following semantic HTML, let's have `<section class="inputs"> ... </section>` instead of `<div class="inputs">`
- [ ] Your inputs to submit a new score is missing a <form> tag. Please wrap your inputs in a tag 🙏